inner-scripts/kubestellar

```diff
@@ -323,10 +372,45 @@ function ensure_espw() {
     else
         kubectl ws create "$espw_name" --enter
     fi
+    run_dex
+    sleep 2
+    run_kb_backend
+    kubectl apply -f "$bindir/../config/crds/"
+    kubectl get crd -oname | grep kubestellar.io | xargs -i kubectl label {} kube-bind.io/exported=true
     eval kubectl apply -f "$bindir/../config/exports" $verbdir
     echo "Finished populating the espw with kubestellar apiexports"
 }
 
+function run_dex() {
+    echo "--< starting dex in background >--"
+    dex serve "$bindir/../dex/dex-config-dev.yaml" &
```

Comment:
Where is `$bindir/../dex` coming from? I do not see a `dex` directory in this branch.

Reply:
This dir is created by a docs-ecutable file during the CI test.

Reply:
The `kubestellar` script needs to work for users and contributors too.

Reply:
I agree. But the docs-ecutable is also going to be followed by users or contributors, right?

Reply:
The `kubestellar` script is intended to be usable directly by contributors, not only through a docs-ecutable document.

Reply:
OK. Which doc, in addition to the docs-ecutable, is going to be followed by a human in order to ensure kcp and kcp plugins to be available? I think I should add similar steps for dex into that doc as well.

Reply:
The commands.md file has an entry for the `kubestellar` script and that should talk about any exceptional requirements for the user's `$PATH` that are not already covered by the introductory material higher in the outline.

I see that example1 tells the user to get the dex bin directory on their `$PATH`. That is the more ordinary approach, rather than expecting one bin dir to be in a certain position relative to another.

Reply:
Sorry, I lost the thread a bit. Let me try again.

kube-bind already comes with a fixed layout, with sibling bin/ and hack/ directories.  I see that the `dex-config-dev.yaml` file originates in kube-bind's hack/ directory. Given that we are already telling users to fetch the kube-bind repo and build its bin and put that on their `$PATH`, I think the simplest thing for our scripts to do is expect that `hack/dex-config-dev.yaml` can be found adjacent to the `$PATH` directory containing `kubectl-bind`.

Reply:
Changed in https://github.com/kubestellar/kubestellar/pull/1340.

Reply:
done